staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go

```diff
@@ -209,3 +214,30 @@ func (r *MaxOfRateLimiter) Forget(item interface{}) {
 		limiter.Forget(item)
 	}
 }
+
+// WithMaxWaitRateLimiter have maxDelay which avoids waiting too long
+type WithMaxWaitRateLimiter struct {
+	limiter  RateLimiter
+	maxDelay time.Duration
+}
+
+func NewWithMaxWaitRateLimiter(limiter RateLimiter, maxDelay time.Duration) RateLimiter {
+	return &WithMaxWaitRateLimiter{limiter: limiter, maxDelay: maxDelay}
+}
+
```

Comment:
```suggestion

```

This method is the same as `WithMaxWait`, right?

Reply:
it is duplicate, i will remove WithMaxWait function